File: ver_cpp/T10.cpp
// https://leetcode.cn/problems/swap-numbers-lcci/
#include <bits/stdc++.h>
using namespace std;

class Solution {
public:
    vector<int> swapNumbers(vector<int>& numbers) {
        vector<int> ans;
        ans.push_back(numbers[1]);
        ans.push_back(numbers[0]);
        return ans;
    }
};

int main(){
    Solution s;
    vector<int> numbers;
    int n;
    cin >> n;
    numbers.push_back(n);
    cin >> n;
    numbers.push_back(n);
    vector<int> ans = s.swapNumbers(numbers);
    cout << ans[0] << "," << ans[1] << endl;
    return 0;
}

File: ver_py/T10.py
# https://leetcode.cn/problems/swap-numbers-lcci/
from typing import List
class Solution:
    def swapNumbers(self, numbers: List[int]) -> List[int]:
        numbers[0], numbers[1] = numbers[1], numbers[0]
        return numbers

if __name__ == '__main__':
    numbers = [int(x) for x in input().split()]
    print(Solution().swapNumbers(numbers))